### ISSUE TITLE
PHPDoc generates @returns instead of @return

### ISSUE THREAD

**Issue by Naatan** [davestewart/komodo-autocode#1]
The proper keyword is @return, not @returns, which is what is used right now when generating phpdoc on a method.

**Comment by davestewart** [comment]
Looks like either is acceptable, but I'll look at this when I tag a release version. 

http://manual.phpdoc.org/HTMLframesConverter/default/phpDocumentor/tutorial_tags.return.pkg.html

Is it causing problems at the moment?

**Comment by Naatan** [comment]
No it's not, I just noticed it when I started using your plugin and couldn't find any reference to @returns in the documentation on the phpdoc website. I haven't actually tried generating the phpdoc yet so as you say it may very well be acceptable.

That said, it'd be nice if there was a way to configure what alias to use, seeing as it'll cause slightly messy phpdoc in a project with multiple devs.

Just a small annoyance, nothing major.
